<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>598</width>
    <height>488</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Folder Pics/folder.png</normaloff>../Folder Pics/folder.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTreeWidget" name="treeWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>441</width>
      <height>441</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Folder</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignLeading|AlignVCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignLeading|AlignVCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Date</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignLeading|AlignVCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Type</string>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="textAlignment">
      <set>AlignLeading|AlignVCenter</set>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>370</y>
      <width>121</width>
      <height>70</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QPushButton" name="apply">
       <property name="font">
        <font>
         <family>Ubuntu Condensed</family>
         <italic>true</italic>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
color: #333;
border: 2px solid #555;
border-radius: 20px;
border-style: outset;
background: qradialgradient(
cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 #888
);
padding: 5px;
}

QPushButton:hover {
background: qradialgradient(
cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 #bbb
);
}</string>
       </property>
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancel">
       <property name="font">
        <font>
         <family>Ubuntu Condensed</family>
         <italic>true</italic>
        </font>
       </property>
       <property name="whatsThis">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;QPushButton {&lt;/p&gt;&lt;p&gt;color: #333;&lt;/p&gt;&lt;p&gt;border: 2px solid #555;&lt;/p&gt;&lt;p&gt;border-radius: 20px;&lt;/p&gt;&lt;p&gt;border-style: outset;&lt;/p&gt;&lt;p&gt;background: qradialgradient(&lt;/p&gt;&lt;p&gt;cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,&lt;/p&gt;&lt;p&gt;radius: 1.35, stop: 0 #fff, stop: 1 #888&lt;/p&gt;&lt;p&gt;);&lt;/p&gt;&lt;p&gt;padding: 5px;&lt;/p&gt;&lt;p&gt;}&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;p&gt;QPushButton:hover {&lt;/p&gt;&lt;p&gt;background: qradialgradient(&lt;/p&gt;&lt;p&gt;cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,&lt;/p&gt;&lt;p&gt;radius: 1.35, stop: 0 #fff, stop: 1 #bbb&lt;/p&gt;&lt;p&gt;);&lt;/p&gt;&lt;p&gt;}&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton {
color: #333;
border: 2px solid #555;
border-radius: 20px;
border-style: outset;
background: qradialgradient(
cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 #888
);
padding: 5px;
}

QPushButton:hover {
background: qradialgradient(
cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 #bbb
);
}</string>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>595</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(85, 87, 83)</string>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>0</y>
      <width>3</width>
      <height>440</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(85, 87, 83)</string>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="step2">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>170</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
color: #333;
border: 2px solid #555;
border-radius: 20px;
border-style: outset;
background: qradialgradient(
cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 #888
);
padding: 5px;
}

QPushButton:hover {
background: qradialgradient(
cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 #bbb
);
}</string>
    </property>
    <property name="text">
     <string>Phase2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="step3">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>220</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
color: #333;
border: 2px solid #555;
border-radius: 20px;
border-style: outset;
background: qradialgradient(
cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 #888
);
padding: 5px;
}

QPushButton:hover {
background: qradialgradient(
cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 #bbb
);
}</string>
    </property>
    <property name="text">
     <string>Phase3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="step4">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>270</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
color: #333;
border: 2px solid #555;
border-radius: 20px;
border-style: outset;
background: qradialgradient(
cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 #888
);
padding: 5px;
}

QPushButton:hover {
background: qradialgradient(
cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 #bbb
);
}</string>
    </property>
    <property name="text">
     <string>phase4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="log">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>120</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
color: #333;
border: 2px solid #555;
border-radius: 20px;
border-style: outset;
background: qradialgradient(
cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 #888
);
}
QPushButton:hover {
background: qradialgradient(
cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 #bbb
);
}</string>
    </property>
    <property name="text">
     <string>Application Log</string>
    </property>
   </widget>
   <widget class="QPushButton" name="add">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>20</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <weight>50</weight>
      <italic>true</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
color: #333;
border: 2px solid #555;
border-radius: 20px;
border-style: outset;
background: qradialgradient(
cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 #888
);
}
QPushButton:hover {
background: qradialgradient(
cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 #bbb
);
}</string>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="remove">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>70</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu Condensed</family>
      <italic>true</italic>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
color: #333;
border: 2px solid #555;
border-radius: 20px;
border-style: outset;
background: qradialgradient(
cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 #888
);
}
QPushButton:hover {
background: qradialgradient(
cx: 0.3, cy: -0.4, fx: 0.3, fy: -0.4,
radius: 1.35, stop: 0 #fff, stop: 1 #bbb
);
}</string>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>598</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOption">
    <property name="title">
     <string>Option</string>
    </property>
    <widget class="QMenu" name="menuSort">
     <property name="title">
      <string>Sort by</string>
     </property>
     <addaction name="actionName"/>
     <addaction name="separator"/>
     <addaction name="actionDate"/>
     <addaction name="separator"/>
     <addaction name="actionType"/>
     <addaction name="separator"/>
     <addaction name="actionReset"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="menuSort"/>
    <addaction name="separator"/>
    <addaction name="actionUnizp_file"/>
    <addaction name="separator"/>
    <addaction name="actionRefresh"/>
   </widget>
   <widget class="QMenu" name="menuAbout_Us">
    <property name="title">
     <string>About Us</string>
    </property>
    <addaction name="actionHelp"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionPrivacy_policy"/>
   </widget>
   <widget class="QMenu" name="menuMemberShip">
    <property name="title">
     <string>MemberShip</string>
    </property>
    <addaction name="actionWhy_should_we_join_FMS_community"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuOption"/>
   <addaction name="menuAbout_Us"/>
   <addaction name="menuMemberShip"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionName">
   <property name="icon">
    <iconset resource="Image.qrc">
     <normaloff>:/IMAGE1/Folder Pics/sort.png</normaloff>:/IMAGE1/Folder Pics/sort.png</iconset>
   </property>
   <property name="text">
    <string>Name</string>
   </property>
  </action>
  <action name="actionDate">
   <property name="icon">
    <iconset resource="Image.qrc">
     <normaloff>:/IMAGE1/Folder Pics/sort.png</normaloff>:/IMAGE1/Folder Pics/sort.png</iconset>
   </property>
   <property name="text">
    <string>Date</string>
   </property>
  </action>
  <action name="actionType">
   <property name="icon">
    <iconset resource="Image.qrc">
     <normaloff>:/IMAGE1/Folder Pics/sort.png</normaloff>:/IMAGE1/Folder Pics/sort.png</iconset>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="icon">
    <iconset>
     <normaloff>../Folder Pics/help.png</normaloff>../Folder Pics/help.png</iconset>
   </property>
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionPrivacy_policy">
   <property name="icon">
    <iconset resource="Image.qrc">
     <normaloff>:/IMAGE1/Folder Pics/priavcy.png</normaloff>:/IMAGE1/Folder Pics/priavcy.png</iconset>
   </property>
   <property name="text">
    <string>Privacy and policy</string>
   </property>
  </action>
  <action name="actionWhy_should_we_join_FMS_community">
   <property name="text">
    <string>Why should we join FMS community ?</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="icon">
    <iconset resource="Image.qrc">
     <normaloff>:/IMAGE1/Folder Pics/reset2.png</normaloff>:/IMAGE1/Folder Pics/reset2.png</iconset>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
  </action>
  <action name="actionUnizp_file">
   <property name="text">
    <string>Unizp file</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="Image.qrc">
     <normaloff>:/IMAGE1/Folder Pics/refresh2.png</normaloff>:/IMAGE1/Folder Pics/refresh2.png</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Image.qrc"/>
 </resources>
 <connections/>
</ui>
